### DISCUSSION TITLE
アドベントカレンダーのカレンダーランキング

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#576] [要望・アイデア・不具合]
アドベントカレンダーのカレンダーランキングから[プログラミング言語](https://qiita.com/advent-calendar/2023/ranking/feedbacks/categories/programming_languages)を見ていて、以下気になったのでご教示ください。

このdiscussionを書いている時点で、Pythonがいいね数順と、購読数順にそれぞれランキングされていません。
カレンダー一覧の[プログラミング言語](https://qiita.com/advent-calendar/2023/categories/programming_languages)からPythonの数字を見ると、ランキングされても良いように見えたので、ランキングが正しく機能しているのか気になりました。ご確認お願いします。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご報告ありがとうございます。
不具合の可能性があるため、原因を調査して修正いたします🙇

**Reply by tanaka-asami415** [reply]
@Brutus03 
こちら修正完了いたしました！現在はランキングに表示されています！

スパム防止の一環のため詳細はお伝えできないのですが、今後同様の事が発生した際、すぐに気づけるよう修正を行う予定です。
この度はご報告いただきありがとうございました🙇

**Reply by Brutus03** [reply]
修正確認しました。
クローズで大丈夫です。ご対応ありがとうございました。
